createrepo_c.spec

```diff
@@ -12,8 +12,10 @@
 
 %if 0%{?rhel}
 %bcond_with zchunk
+%bcond_with zstd
```

Comment:
Unlike zchunk, support for zstd compressed metadata is (I believe) already enabled in libsolv on RHEL.  There's just no convenient way to generate the repos yet.  So it might make sense to likewise default to ON for RHEL >7.  Or at least 9+.

Reply:
Ok, I have enabled it also for rhel 8. I believe it was enabled by: https://bugzilla.redhat.com/show_bug.cgi?id=1914876 so I have added a note about that.